tests/integration/misc-test.js

```diff
@@ -0,0 +1,62 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { get, set } from '@ember/object';
+import { Changeset } from 'ember-changeset';
+import hbs from 'htmlbars-inline-precompile';
+import { render, find, settled } from '@ember/test-helpers';
+
+module('Integration | misc', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it re-renders derived state automatically when root state changes', async function(assert) {
+    class MyModel {
+      isOptionOne = false;
+      isOptionTwo = false;
+      isOptionThree = false;
```

Comment:
What if you add `@tracked` to these properties?

Reply:
Sorry, missed that.
It still can be reproduced with `@tracked`